pkgs/applications/networking/browsers/firefox/common.nix

```
@@ -169,10 +169,8 @@ buildStdenv.mkDerivation ({
 
   inherit src unpackPhase meta;
 
-  outputs = [
-    "out"
-    "symbols"
-  ];
+  outputs = [ "out" ]
```

```suggestion
  outputs = [
    "out"
  ]
```

To match the list style everywhere else.